umbrui.py

```diff
@@ -28,10 +29,6 @@ def __init__(self):
         os.putenv("SDL_AUDIODRIVER", "dummy")
 
         self.init_screen()
-        
-        # Init RPC connections
-        self.btc_grpc = BtcRPC()
```

Comment:
Why should these not be here?

Reply:
Because we don't need them here anymore, all the connection stuff is done outside it, but I'll change it.